tests/test_hf_api.py

```diff
@@ -229,31 +223,28 @@ def test_delete_repo_missing_ok(self) -> None:
         self._api.delete_repo("repo-that-does-not-exist", missing_ok=True)
 
     def test_create_update_and_delete_repo(self):
-        REPO_NAME = repo_name("crud")
-        self._api.create_repo(repo_id=REPO_NAME)
-        res = self._api.update_repo_visibility(repo_id=REPO_NAME, private=True)
-        self.assertTrue(res["private"])
-        res = self._api.update_repo_visibility(repo_id=REPO_NAME, private=False)
-        self.assertFalse(res["private"])
-        self._api.delete_repo(repo_id=REPO_NAME)
+        repo_id = self._api.create_repo(repo_id=repo_name()).repo_id
```

Comment:
what is the diff between using "crud" or not as arg?

Reply:
`repo_name` is generating a string to use as repo name. Passing `"crud"` will had `"crud"` to the repo name. It might have been useful at some point for debugging but now it's more confusing that anything so better to just remove it.